pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -1914,6 +1914,55 @@ var _ = Describe("Converter", func() {
 			Expect(Convert_v1_VirtualMachineInstance_To_api_Domain(vmi, domain, c)).To(Succeed())
 			Expect(domain.Spec.Devices.HostDevices).To(Equal([]api.HostDevice{{Type: identifyDevice}}))
 		})
+
+		It("should successfully passthrough a mediated device with a disabled display", func() {
+			vmi.Spec.Domain.Resources.Requests = k8sv1.ResourceList{
+				k8sv1.ResourceMemory: resource.MustParse("1G"),
+			}
```

Comment:
why do we need this?

Reply:
It's the same as what was in the e2e test, so I assumed it is needed. is it not?